### PR TITLE
Indexing and protobuf fixes

### PR DESCRIPTION
**Pull request by anistor**

### CODE REVIEW

File: annotated-pojos/src/main/java/annotated/pojos/Pokemon.java [hunk 1]
```diff
@@ -6,9 +6,11 @@
 @ProtoDoc("@Indexed")
 public class Pokemon {
 
+   @ProtoDoc("@Field")
```
Comment (galderz) [inline annotated-pojos/src/main/java/annotated/pojos/Pokemon.java]: Why add this?

File: annotated-pojos/src/test/java/annotated/pojos/PokemonQueryTest.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public void testPokemonQuery() {
 
       QueryFactory queryFactory = Search.getQueryFactory(cache);
 
-      Query query = queryFactory.create("FROM annotated.pojos.Pokemon p where p.type1 = :type");
+      Query query = queryFactory.create("FROM pokemons.Pokemon p where p.type1 = :type");
```
Comment (galderz) [inline annotated-pojos/src/test/java/annotated/pojos/PokemonQueryTest.java]: Why change the package name? To make a separation between java package names and proto package names?

File: plain-pojos/src/main/resources/crypto.proto [hunk 3]
```diff
@@ -1,9 +1,18 @@
 package plain.pojos;
 
+/**
```
Comment (galderz) [inline plain-pojos/src/main/resources/crypto.proto]: Why these? Works just fine without them

File: annotated-pojos/src/main/java/annotated/pojos/Pokemon.java (hunk 1 above)
Comment (anistor) [reply annotated-pojos/src/main/java/annotated/pojos/Pokemon.java]: It's explained in the commit comment.

File: annotated-pojos/src/test/java/annotated/pojos/PokemonQueryTest.java (hunk 2 above)
Comment (anistor) [reply annotated-pojos/src/test/java/annotated/pojos/PokemonQueryTest.java]: yes. also explained in the commit comment.

File: plain-pojos/src/main/resources/crypto.proto (hunk 3 above)
Comment (anistor) [reply plain-pojos/src/main/resources/crypto.proto]: It causes the fileds to be indexed, otherwise it works but does not use index.

Comment (galderz) [reply plain-pojos/src/main/resources/crypto.proto]: Is there a WARN message or something for such scenarios? I can see this being a very easy error to make by users.

**Comment by galderz** [comment]
Thx @anistor, integrating :)

File: plain-pojos/src/main/resources/crypto.proto (hunk 3 above)
Comment (anistor) [reply plain-pojos/src/main/resources/crypto.proto]: I have an ongoing PR for detecting such annotation missuse and there will be warnings in ISPN 9.3

**Pull request merged**